### PR TITLE
Backport arm64/riscV support

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**
Backport support from arm64 and riscv64 (71562720ac3641e61b7814618f407f4de7a94bca) and update go-efilib (proposed in https://github.com/snapcore/secboot/pull/208), which contains also changes affecting arm64/riscv64.

### CODE REVIEW

**Comment by pedronis** [comment]
blocked until the corresponding code lands on master first

**Comment by alfonsosanchezbeato** [comment]
@pedronis #208 has been merged now

**Review by pedronis — APPROVED** [review]
thanks

**Pull request merged**
